src/components/Facility/CentralNursingStation.tsx

```diff
@@ -85,6 +85,7 @@ export default function CentralNursingStation({ facilityId }: Props) {
       noImplicitPadding
       breadcrumbs={false}
       collapseSidebar
+      className="@container"
```

Comment:
@rithviknishad ,@nihal  ,this   className="@container"  causing this bug https://github.com/ohcnetwork/care_fe/pull/9013/#issuecomment-2473293072

Reply:
however, like this creating a new parent for this it will not cause this bug and work properly 
![Screenshot 2024-11-14 221551](https://github.com/user-attachments/assets/acd32884-dbda-4e6d-a3c0-434bb5899ef4)